src/components/ApplicationGroup/AppGroupDetailsRoute.tsx

```diff
@@ -15,29 +15,68 @@ import { EMPTY_LIST_MESSAGING, ENV_APP_GROUP_GA_EVENTS, NO_ACCESS_TOAST_MESSAGE
 import { ReactComponent as Settings } from '../../assets/icons/ic-settings.svg'
 import { getEnvAppList } from './AppGroup.service'
 import { AppGroupAdminType, EnvHeaderType } from './AppGroup.types'
+import { getAppList } from '../app/service'
+import { MultiValue } from 'react-select'
+import { OptionType } from '../app/types'
+import AppGroupAppFilter from './AppGroupAppFilter'
 import EnvCIDetails from './Details/EnvCIDetails/EnvCIDetails'
 import EnvCDDetails from './Details/EnvCDDetails/EnvCDDetails'
 import '../app/details/app.css'
 
+const AppGroupAppFilterContext = React.createContext(null)
+
+export function useAppGroupAppFilterContext() {
+    const context = React.useContext(AppGroupAppFilterContext)
+    if (!context) {
+        throw new Error(`cannot be rendered outside the component`)
```

Comment:
Can we move this error message in constant messaging

Reply:
Done